.github/workflows/ci-linkchecks.yml

```diff
@@ -0,0 +1,29 @@
+name: Linkcheck
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 06 * * 1-5"
```

Comment:
Do you think running at the weekend too would give valuable extra information about the persistence of some link problems?

>It's been failing for 3 mornings in a row, must be a real problem

Reply:
Maybe does no harm, but we might end up with multiple issues stacking up?

@trexfeathers @pp-mo Decided: let's just try it

Reply:
See https://github.com/SciTools/iris/pull/6386/commits/1da5538cf126b4f3f386fdeb3c59fa97df6c89b5